src/components/ProgramDrawer.tsx

```diff
@@ -168,11 +170,19 @@ export default function ProgramDrawer({ setEQIPChecked }): JSX.Element {
         >
             <Box sx={{ height: 100 }} />
             <MenuItem style={{ whiteSpace: 'normal' }} sx={{ my: 1 }}>
```

Comment:
```suggestion
            <MenuItem disabled style={{ whiteSpace: 'normal' }} sx={{ my: 1 }}>
```
could be a better option here as it disables the entire sub-menu.

Reply:
Thanks! I think the menu items should be functional in near future, so would it be better to change their styles in this way?